webserver/src/models/user.rs

```diff
@@ -1,4 +1,4 @@
-use crypto::{digest::Digest, md5::Md5};
+use md5::{Md5, Digest};
```

Comment:
This is super nitpicky, but I this import should be between `failure` and `rocket`.

I like to go alphabetical crates, local crate, std as the order.

It should also be in alphabetical order itself, as below:

```suggestion
use md5::{Digest, Md5};
```